src/helics/apps/Player.cpp

```diff
@@ -774,16 +774,28 @@ namespace apps {
 static int hasB64Wrapper(const std::string& str)
 {
     if (str.front() == '\"') {
+        if (str.size() < 8) {
+            return 0;
+        }
         if ((str.compare(2, 3, "64[") == 0) && (str[str.size() - 2] == ']')) {
             return 5;
         }
+        if (str.size() < 11) {
+            return 0;
+        }
         if ((str.compare(5, 3, "64[") == 0) && (str[str.size() - 2] == ']')) {
             return 8;
         }
```

Comment:
in JSON it always needs to be quoted.   I was trying to keep it consistent between the two to the extent possible.  Basically treating the input as a valid JSON value in the text file.  (but assuming a string otherwise)  On write it always writes a valid JSON string value,